lib/linguist/languages.yml

```diff
@@ -1099,6 +1099,15 @@ DTrace:
   codemirror_mode: clike
   codemirror_mime_type: text/x-csrc
   language_id: 85
+Dafny:
+  type: programming
+  extensions:
+  - ".dfy"
+  interpreters:
+  - dafny
+  tm_scope: text.dfy.dafny
+  ace_mode: text
+  language_id: 969323346
```

Comment:
I see you've not specified a colour for the language. Is this intentional? Without an explicit colour, the language will show up using the default `#cccccc` on some views like the language bar and with no colour at all on others like the repo listing.

Reply:
No, I didn't include it only because it wasn't strictly required, but I can propose a color. Is there a place I can view all currently used colors, or render them from the library?

Reply:
Unfortunately not as it's not simply a matter of picking a free colour as the colour also can't be too similar to another already in use. https://github.com/github/linguist/blob/master/test/test_color_proximity.rb is the test that enforces the proximity.  I'm trying to get the GitHub designers to allow me to drop this but my request is in the backlog of things to consider 😞.

The best thing to do is select a colour from the language's branding (eg logo, website, docs) and then run that test and keep tweaking the colour until you no longer get an error.

By starting with a colour from the branding, you also reduce the chances of someone else wanting to change it on a whim and thus have be directed to the requirements detailed at https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-color-associated-with-a-language for changing the colour.

Reply:
Thanks! I managed to wedge our preferred color in-between Javascript and Isabelle with minor not-human-perceptible tweaking. :)